src/test/kotlin/uk/gov/justice/digital/hmpps/incentivesapi/integration/wiremock/PrisonApiMockServer.kt

```diff
@@ -430,11 +430,15 @@ class PrisonApiMockServer : WireMockServer(WIREMOCK_PORT) {
 
   fun stubPositiveCaseNoteSummary() {
     stubFor(
-      post("/api/case-notes/usage").willReturn(
-        aResponse()
-          .withHeader("Content-Type", "application/json")
-          .withBody(
-            """
+      post("/api/case-notes/usage")
+        .withRequestBody(
+          WireMock.equalToJson("""{"numMonths": 3, "type": "POS"}""", true, true)
```

Comment:
Added `RequestBody` matching otherwise this function and `stubNegativeCaseNoteSummary` could not be differentiated by WireMock as they stubbed the same endpoint